types/algoliasearch/index.d.ts

```diff
@@ -1099,120 +1100,121 @@ declare namespace algoliasearch {
     userToken?: string;
   }
   interface QueryParameters {
+   /**
+    * Query string used to perform the search
+    * default: ''
+    * https://www.algolia.com/doc/api-reference/api-parameters/query/
+    */
+   query?: string;
+   /**
+    * Filter the query with numeric, facet or/and tag filters
+    * default: ""
+    * https://www.algolia.com/doc/api-reference/api-parameters/filters/
+    */
+   filters?: string;
+   /**
+    * A string that contains the list of attributes you want to retrieve in order to minimize the size of the JSON answer.
+    * default: *
+    * https://www.algolia.com/doc/api-reference/api-parameters/attributesToRetrieve/
+    */
+   attributesToRetrieve?: string[];
+   /**
+    * List of attributes you want to use for textual search
+    * default: attributeToIndex
+    * https://www.algolia.com/doc/api-reference/api-parameters/restrictSearchableAttributes/
+    */
+   restrictSearchableAttributes?: string[];
+   /**
+    * You can use facets to retrieve only a part of your attributes declared in attributesForFaceting attributes
+    * default: []
+    * https://www.algolia.com/doc/api-reference/api-parameters/facets/
+    */
+   facets?: string[];
```

Comment:
seems good to restrict this to only array form, we still also support comma separated string, but that makes the typing confusing.

Reply:
good like you changed it to me

Reply:
Alright, thanks!